java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerSerializationIT.java

```diff
@@ -52,24 +53,26 @@
 
     private ConfigContext config;
 
-    @BeforeClass()
-    @Parameters({"usingEncryption"})
-    public void beforeClass(@org.testng.annotations.Optional Boolean usingEncryption) throws IOException {
+    @Parameters({"encryptionCipher"})
+    public EncryptedServerSideMultipartManagerSerializationIT(final @Optional String encryptionCipher) throws IOException {
```

Comment:
A ```compilation-error``` will be generated here since this ```integration-test``` will mandatorily require invocation of the TestNG parameter ```encryptionCipher```